components/Cards/index.js

```diff
@@ -38,6 +48,16 @@ test1.forEach(e => {
 // axios.get(`https://lambda-times-backend.herokuapp.com/articles`)
```

Comment:
i like how you comment out things that don't work and keep them there as reference